modules/dr-restoring-cluster-state.adoc

```diff
@@ -299,6 +299,17 @@ Perform the following step only if you are using `OVNKubernetes` network plugin.
 $ oc delete node <non-recovery-controlplane-host-1> <non-recovery-controlplane-host-2>
 ----
 
+. Verify that the Cluster Network Operator (CNO) re-deploys the OVN-Kubernetes control plane, and it no longer references the wrong controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before proceeding with the next step.
```

Comment:
```suggestion
. Verify that the Cluster Network Operator (CNO) redeploys the OVN-Kubernetes control plane and that it no longer references the wrong controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before you proceed with the next step.
```

Reply:
I checked the ISG "Prefixes" topic (https://www.ibm.com/docs/en/ibm-style?topic=grammar-prefixes), and based on the guidance there, I think "redeployed" is correct.